processor/spanmetricsprocessor/README.md

```diff
@@ -78,7 +78,7 @@ processors:
   batch:
   spanmetrics:
     metrics_exporter: otlp/spanmetrics
-    latency_histogram_buckets: [2ms, 6ms, 10ms, 100ms, 250ms]
+    latency_histogram_buckets: [5ns, 8us, 2ms, 6ms, 10ms, 100ms, 250ms]
```

Comment:
ns is too low, I would just add 100us and 1ms initially

Reply:
Also I am not sure, is this consider breaking change in prom?

Reply:
> ns is too low, I would just add 100us and 1ms initially

Sure, that's done.

> Also I am not sure, is this consider breaking change in prom?

Do you mean if this precision is supported by prom functions? If so, I tested it on `histogram_quantile` and seems to be working as expected when I submit sub-millisecond latencies. Otherwise, please clarify what is meant by breaking change in prom.

<img width="1045" alt="Screen Shot 2021-07-03 at 5 56 11 pm" src="https://user-images.githubusercontent.com/26584478/124347544-9a45cb80-dc28-11eb-81c7-ebd980193ccf.png">

Reply:
If someone has already a graph with the current metric, adding the new boundaries would that break them?

Reply:
Oh I see what you mean, this specific change is just in the README (and also in another sample config) to suggest to readers that the spanmetrics processor now supports sub-millisecond time buckets, it does not break existing latency buckets because I have not altered the [default latency buckets](https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/4091/files#diff-78f9e7446e0e4fb474786b4b2c262486887bf27479f1f8bc598376d6c3c29b72R49).